README.rst

```diff
@@ -39,7 +39,15 @@ maximum number of times you'd like the tests to run:
 
 .. code-block:: bash
 
-  $ py.test --rerun 5
+  $ pytest --reruns 5
+
+To add a delay time between re-runs use the ``--reruns-delay`` command line
+option with amount of seconds that you would like wait before the next test
+re-run is lunched:
```

Comment:
:hamburger: :smile: 

Pretty sure you mean _launched_!

Reply:
:) Probably I was very hungry at the time when writing that. How should I go about fixing that, new PR?

Reply:
Haha. No need for a new PR, just push another commit to your branch and it'll show up here.